src/Razor/tsconfig.json

```diff
@@ -7,12 +7,18 @@
     "strict": true,
     "moduleResolution": "node",
     "noUnusedLocals": true,
+    "skipDefaultLibCheck": true,
```

Comment:
Do we still need the `tsconfig` for `eslint`?

Reply:
Not sure I follow, but if the question is "do we still need tsconfig files once we switch to eslint" the answer is yes. tsconfig files are used by the TypeScript compiler to set things like output directory and compilation options, whereas eslint is a separate process which only lints the ts files. If that wasn't the question could you re-phrase it?

Reply:
Ah okay, thanks for the clarification. I interpreted `tsconfig` as the config file for `tslint`.